README.rst

```diff
@@ -24,7 +24,33 @@ includes:
 .. _NAMD: http://www.ks.uiuc.edu/Research/namd/
 .. _`other simulation codes`: https://alchemlyb.readthedocs.io/en/latest/parsing.html
 .. _`pymbar`: http://pymbar.readthedocs.io/
-   
+
+
+Installation
+------------
+
+**Install** via ``pip`` from `PyPi (alchemlyb)`_ ::
+
+  pip install alchemlyb
+
+or as a `conda`_ package from the `conda-forge (alchemlyb)`_ channel
+::
+
+  conda install -c conda-forge alchemlyb 
+
+
+**Update** with ::
+
+  pip install --update alchemlyb
+
+when using ``pip`` or with ``conda`` run ::
```

Comment:
Might be good to separate this with
```
when using ``pip`` ::
    pip install --update alchemlyb
when using ``conda`` ::
    conda update -c conda-forge alchemlyb
```

Reply:
Yes, I agree, even though the current version was grammatically correct, it wasn't clear. Fixed.